Created Requirements.md

TuringsRevenge opened the pull request:
Added requirements to the document regarding basic functionality.

TuringsRevenge committed:
Created Requirements.md

Added requirements to the document regarding basic functionality.

TuringsRevenge committed:
Updated Requirements

Defined the numbered user stories more precisely and changed 3.2)

swartzy4 reviewed (commented):
I'm not a fan of requirement 3.2 only because we don't know how big the user's display will be. I think we should make an assumption that the user's display will not surpass x inches (haven't thought of an exact measurement yet).

TuringsRevenge committed:
Updated 3.2. 

Changed the definition of the space taken up by using a percentage of the screen instead of a specific measurement.

TuringsRevenge committed:
Created features.md 

Ignore old "features.md" pull request. This is now the new features document since I'm committing it to the newly created rdv_DynamicShadows branch.

doorz1009 commented on Features.md:
```diff
@@ -0,0 +1,17 @@
+# Feature
```
This file is being added in two different branches, this will cause a conflict when merged. Either add this file in this branch or the other branch. But, do not do this in both branches.

TuringsRevenge committed:
Updated features.md 

Changed the wording of the name of the game because using "MineCraft" would violate a copyright issue. Updated the user story about trees and plants casting shadows because the current wording was not sufficient enough to fully express what exactly was to be done.
